src/app/components/dashboard-apps-dock/dashboard-apps-dock.html

```diff
@@ -0,0 +1,87 @@
+                <a href="" ng-click="helper.oAuthConnectPath(app)"><i class="fa fa-play-circle" >
+                </i> {{ 'mno_enterprise.templates.dashboard.apps_list.connect' | translate }}</a>
+            </div>
+            <div class="menu-item ng-hide" ng-show="helper.displaySettings()">
+              <a ng-click="showSettingsModal(app)"><i class="fa fa-cog"></i> Settings</a>
+            </div>
+          </div>
+        </div>
+      </li>
+      <li>
+        <div class="add-app">
+          <a ui-sref="home.marketplace">
+            <i class="fa fa-plus-circle"></i>
+            <span>Add&nbsp;more apps</span>
+          </a>
+        </div>
+      </li>
+      <li class="expand-menu">
+        <div>
+          <a ng-click="appDock.toggle()" ng-switch="appDock.isMinimized">
+            <i class="fa fa-chevron-down" ng-switch-when="true"></i>
+            <i class="fa fa-chevron-up" ng-switch-when="false"></i>
+          </a>
+        </div>
+      </li>
+    </ul>
+  </div>
+</div>
+
+<div id="app-dock-mobile-container" class="visible-xs" ng-if="Object.keys(apps).length > 0 || visibleIfEmpty == true">
+  <div class="mobile-dock align-center btn" ng-click="launchApp.isClosed = !launchApp.isClosed">
+    Launch App
+    <i class="fa fa-chevron-down pull-right" ng-hide="launchApp.isClosed"></i>
+  </div>
+  <div id="mobile-dock" collapse="launchApp.isClosed">
+    <ul class="list-unstyled">
+      <li ng-repeat="app in apps" ng-class="{active: isActiveApp(app.id)}" ng-switch on="isActiveApp(app.id)">
+        <div class="app-line" ng-switch-when="true">
+          <i class="fa fa-angle-left" ng-click="setActiveApp($event, app.id)"></i>
+          <div ng-show="app.tutorial">
+            <a class="btn btn-shaded" ng-href="{{app.tutorial}}"><span class="fa fa-book"></span> Tutorials</a>
+            <br>
+          </div>
+          <div ng-show="helper.displaySettings()">
+            <a class="btn btn-shaded" ng-click="showSettings(app)"><span class="fa fa-cog"></span> Settings</a>
+          </div>
+        </div>
+        <div class="app-line" ng-switch-default>
+          <img class="app-logo" ng-src="{{app.logo}}">
+          <div>
+            <div class="title">{{app.name}}</div>
+            <div ng-hide="appsListHelper.isLaunchHidden(app)">
+              <a ng-href="{{appsListHelper.appActionUrl(app)}}" target="_blank" class="btn btn-warning">Launch App</a>
+            </div>
+          </div>
+          <i class="fa fa-angle-right" ng-click="setActiveApp($event, app.id)"></i>
+        </div>
+      </li>
+      <div ng-if="visibleIfEmpty">
+        <div class="align-center">
+          <a ui-sref="home.marketplace" class="btn add-app">
+            <span class="fa fa-plus"></span><span> Add more apps</span>
+          </a>
+        </div>
+      </div>
+    </ul>
+    <div class="align-center">
+      <button class="btn btn-shaded" ng-click="launchApp.isClosed = true">Cancel</button>
```

Comment:
Could you verify that `btn-shaded` is defined?

I think `btn-shaded` is a custom class, maybe we should use a standard bootstrap class instead.